### ISSUE TITLE
Build instructions fail

### ISSUE THREAD

**Issue by fhriley** [josenk/terraform-provider-esxi#32]
The build instructions don't work for me on Mac OS. I had to make the following change to get them to work. I'm not familiar enough with go to know if this is the correct solution or not.

```
diff --git a/main.go b/main.go
index f3aeae6..51d23be 100644
--- a/main.go
+++ b/main.go
@@ -3,7 +3,7 @@ package main
 import (
        "github.com/hashicorp/terraform/plugin"
        "github.com/hashicorp/terraform/terraform"
-    "terraform-provider-esxi/esxi"
+    "github.com/terraform-providers/terraform-provider-esxi/esxi"
 )

 func main() {
```

**Comment by silasb** [comment]
I think I introduced this.  I develop with Go modules locally.

The correct import is `"github.com/terraform-providers/terraform-provider-esxi/esxi"`

**Comment by josenk** [comment]
fixed in V1.4.1
>https://github.com/josenk/terraform-provider-esxi/releases/tag/1.4.1
